src/components/orders/StatusLabel/index.tsx

```diff
@@ -65,16 +66,17 @@ function getStatusIcon(status: OrderStatus): IconDefinition {
       return faClock
     case 'filled':
       return faCheckCircle
-    case 'canceled':
+    case 'cancelled':
       return faTimesCircle
     case 'open':
+    case 'signature pending':
       return faCircleNotch
   }
 }
 
 function StatusIcon({ status }: DisplayProps): JSX.Element {
   const icon = getStatusIcon(status)
-  const isOpen = status === 'open'
+  const isOpen = ['open', 'signature pending'].includes(status)
```

Comment:
For the first version this is ok.
As a follow up I'd like to align the styles as well (show the key icon, maybe even the shimmer), but not a priority, can come if we have time or after the release.

Reply:
I have added the key icon and a slight shimming effect

Reply:
Oh already? Thanks